web_external/stylesheets/body/challengeLearn.styl

```diff
@@ -0,0 +1,39 @@
+.bg
+  background: url('/static/built/plugins/covalic/extra/img/challenge-jumbotron.jpg') no-repeat center center
+  position fixed
+  width 100%
+  height 350px
+  top 0
+  left 0
+  z-index -1
+
+.jumbotron
+  height 350px
+  color #111
+  background:transparent
+
+.c-learn-main-text
+  padding-top 100px
+
+
+.video-wrapper
+    position relative
+    padding-bottom 56.25%
+    padding-top 25px
+    height 0
+    margin-bottom 1em
+
+.video-wrapper iframe
+  position absolute
+  top 0
+  left 0
+  width 100%
+  height 100%
+
+.page-title
+  font-family "Source Sans Pro",Calibri,Candara,Arial,sans-serif
+  font-weight 300
+  text-align center
+
+.c-contact-header
+  margin-left 200px
```

Comment:
Should we just center this? It looks rather odd on my screen width, like it was supposed to be centered but is off a bit.